libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -1006,6 +1015,13 @@ bool AC_AttitudeControl::ang_vel_to_euler_rate(const Vector3f& euler_rad, const
 Vector3f AC_AttitudeControl::update_ang_vel_target_from_att_error(const Vector3f &attitude_error_rot_vec_rad)
 {
     Vector3f rate_target_ang_vel;
+
+    // Boost Angle P one very rapid throttle changes
+    if (_motors.get_throttle_slew_rate() > AC_ATTITUDE_CONTROL_THR_G_BOOST_THRESH) {
+        float angle_p_boost = constrain_float((_throttle_gain_boost + 1.0f) * (_throttle_gain_boost + 1.0f), 1.0, 4.0);
+        set_angle_P_scale_mult(Vector3f(angle_p_boost, angle_p_boost, 1.0f));
+    }
+
```

Comment:
@andyp1per this concerns me.  Can you explain how this will not affect heli’s. I have to look at this more to understand what unintended affect this could have on heli’s.

Reply:
_throttle_gain_boost is 0 by default so scale should be 1 and so should not affect helis, but now I remember there is some issue with getting random junk if a param is not initialized, so let me make sure that does not happen